tests/MenderAPI/inventory.py

```diff
@@ -51,7 +51,7 @@ def get_devices(self, has_group=None):
         return ret.json()
 
     def get_device(self, device_id):
-        devurl = "%s%s/%s" % (self.get_inv_base_path(), "device", device_id)
+        devurl = "%s%s/%s" % (self.get_inv_base_path(), "devices", device_id)
```

Comment:
we changed to `devices` completely, I got lost, but I trust you.

Reply:
This is an interesting one: it was wrong (the endpoint is `/devices/:id`) but as the only test which uses it expects a 404, it just worked. :) In the new test, though, I need to get the device and it failed because of the wrong URL, and this uncovered the typo.

Reply:
wow! nice one.